test/organisations_api_test.rb

```diff
@@ -106,64 +100,48 @@ def organisation(slug: "test-department")
         .upon_receiving("a request with page 2 params")
         .with(request.merge(query: "page=2"))
         .will_respond_with(response.merge(headers: { "link" => page_two_links }))
-    end
 
-    it "should handle pagination" do
-      response = api_client.organisations
-
-      assert_equal 20, response["results"].count
-      assert_equal 40, response.with_subsequent_pages.count
+      api_client.organisations.with_subsequent_pages.count
```

Comment:
No need to call `.count` here anymore

Reply:
Thanks - I'll look at removing this now. I seem to remember the second HTTP request didn't happen unless I appended both `.with_subsequent_pages` and `.count`, but I'll take another look to confirm.

Reply:
Yes - the `.count` is needed. Removing it causes this failure:

```
  1) Error:
GdsApi::Organisations::fetching a paginated list of organisations#test_0001_handles pagination:
Pact::MockService::VerificationFailedError: Actual interactions do not match expected interactions for mock Collections Organisation API.

Missing requests:
	GET /api/organisations?page=2

See log/collections_organisation_api_mock_service.log for details.
```